modules/social_features/social_event/social_event.install

```diff
@@ -208,3 +208,17 @@ function social_event_update_130000(): ?string {
   // Output logged messages to a related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Set the end date of the event required.
+ */
+function social_event_update_130001(): ?string {
```

Comment:
We don't need `?string` here, should be just `string`.
See [warning](https://github.com/goalgorilla/open_social/actions/runs/8834463030/job/24256354327?pr=3880): 
```
Function social_event_update_130001() never returns null so it can be removed from the return type.
```

Reply:
@ribel Fixed